pac/atsamd51p/Cargo.toml

```diff
@@ -11,7 +11,7 @@ readme = "README.md"
 edition = "2021"
 
 [dependencies]
-cortex-m = "0.7"
+cortex-m = { version = "0.7", features = ["critical-section-single-core"] }
```

Comment:
The PAC should NOT enable the `critical-section-single-core` feature, only the BSP. Otherwise, users of the PAC would not have the option to bring in their own critical section implementation as needed.

Reply:
Ok yeah that makes sense